openapi/components/requestBodies/storefront/PostPayment.yaml

```
@@ -0,0 +1,25 @@
+content:
+  application/json:
+    schema:
+      oneOf:
+        - type: object
+          title: For transaction
```

This title seems incomplete
```suggestion
          title: Payment
```

---

It's actually intentional – operations is PostPayment and there are two ways of posting the payment `For transaction` or `For invoice`. If that makes sense to you